osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -81,10 +81,66 @@ public void SetOsuLogo(OsuLogo logo)
         private readonly ButtonArea buttonArea;
 
         private readonly MainMenuButton backButton;
+        private readonly MainMenuButton settingsButton;
+
+        // public for tests
+        [CanBeNull]
+        public MainMenuButton PlayButton { get; private set; }
```

Comment:
Doesn't need to be exposed for testing - can be exposed with `.ChildrenOfType<MainMenuButton>().First()` or similar.

Reply:
I decided to keep playButton, but make it internal, since it makes the usage in ButtonSystem a bit more clearer, as well as the tests. Additionally it's not the first child, it's somewhere in the middle, since the back and settings button come before it.